apps/console/src/features/applications/data/application-templates/templates/oidc-web-application/create-wizard-help.tsx

```diff
@@ -59,7 +59,7 @@ const OIDCWebApplicationCreateWizardHelp: FunctionComponent<OIDCWebApplicationCr
             <React.Fragment>
                 <Heading as="h5">Authorized URIs</Heading>
                 <p>
-                    The authorized URI determines where the authorization code is sent to
+                    The authorized URIs determine where the authorization code is sent to
```

Comment:
The redirect URLs determine - > The authorized redirect URLs determine

Reply:
Check other places too

Reply:
Fixed in new commit